src/Modules/CalcSetup.lua

```diff
@@ -621,6 +621,36 @@ function calcs.initEnv(build, mode, override, specEnv)
 							env.itemModDB:ScaleAddMod(mod, scale)
 						end
 					end
+				elseif (slotName == "Weapon 1" or slotName == "Weapon 2") and modDB:Flag(nil, "Condition:EnergyBladeActive") then
```

Comment:
Would it not be easier to just use item:ParseRaw(```
test
Energy Blade One Handed
Quality: 20
Sockets: G-G-G
Implicits: 1```)

and add the raw implicit text from there?